awkward-cpp/pyproject.toml

```diff
@@ -91,22 +91,22 @@ pytest -p no:cacheprovider \
 """]
 skip = [
     "pp*win*",
-    "pp*-manylinux_i686",
```

Comment:
We don't build with 32-bit linux by default anymore in cibuildwheel (and we weren't doing it at all here anyway, like in most packages).